.github/workflows/pytest-workflow.yml

```diff
@@ -2,7 +2,7 @@ name: pytest-workflow
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
   pull_request:
-    branches: [dev, master]
+    branches: [dev]
```

Comment:
Why is `master` being removed?  Don't you want the pytests to run on merge to master?

Reply:
-> .github/workflows/pytest-workflow_release.yml

Reply:
there might be some smarter way to do so, but the idea is to test with docker on dev, and add singularity and conda tests on master when we prepare for release

Reply:
okidoks